utility/src/main/java/org/oppia/android/util/locale/LeftAlignedSymbolsSpan.kt

```diff
@@ -0,0 +1,40 @@
+package org.oppia.android.util.locale
+
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.text.style.ReplacementSpan
+
+// Custom span to force LTR alignment for symbols as well
+// Custom span to force LTR alignment for all text including symbols
```

Comment:
This should be a KDOC rather than a comment.

Reply:
Done.